src/sentry/api/endpoints/organization_details.py

```diff
@@ -576,6 +576,10 @@ def get(self, request: Request, organization) -> Response:
         :param string detailed: Specify '0' to retrieve details without projects and teams.
         :auth: required
         """
+        # This param will be used to determine if we should include feature flags in the response
+        include_feature_flags = request.GET.get("include_feature_flags", "0") != "0"
```

Comment:
we will only include ff if `include_feature_flags=1`

Reply:
I don't know if that's the right attribute to use here. From reading that, it would make sense to use that for a adjacent domain object, not additional data for the current domain object.

Reply:
hmm, that also makes sense, what do you think @sentaur-athena?

Reply:
yes @ykamo001 is right. It's a weird case with feature flags because they're not real objects that we're serializing. Please ignore my comment.